prometheus-exporters/netapp-exporter/charts/netapp-harvest-exporter/aggregations/maia.yaml

```diff
@@ -36,19 +86,79 @@ groups:
+
+    - record: netapp_volume_write_throughput:maia
+      expr: sum by (filer, vserver, volume, share_name, share_id, share_type, project_id) (netapp_volume_write_throughput:pre{project_id!=""}) / count by (filer, vserver, volume, share_name, share_id, share_type, project_id) (netapp_volume_write_throughput:pre)
+
+    - record: netapp_volume_read_latency_milliseconds:maia
+      expr: sum by (filer, vserver, volume, share_name, share_id, share_type, project_id) (netapp_volume_read_latency_milliseconds:pre{project_id!=""}) / count by (filer, vserver, volume, share_name, share_id, share_type, project_id) (netapp_volume_read_latency_milliseconds:pre)
+
+    - record: netapp_volume_write_latency_milliseconds:maia
+      expr: sum by (filer, vserver, volume, share_name, share_id, share_type, project_id) (netapp_volume_write_latency_milliseconds:pre{project_id!=""}) / count by (filer, vserver, volume, share_name, share_id, share_type, project_id) (netapp_volume_write_latency_milliseconds:pre)
+
+    #
+    # fall back to old metrics
+    #
+
+    - record: netapp_volume_total_bytes:maia
+      expr: netapp_volume_total_bytes:maia2 or netapp_volume_total_bytes:maia1{project_id!=""}
+
+    - record: netapp_volume_used_bytes:maia
+      expr: netapp_volume_used_bytes:maia2 or netapp_volume_used_bytes:maia1{project_id!=""}
+
+    - record: netapp_volume_available_bytes:maia
+      expr: netapp_volume_available_bytes:maia2 or netapp_volume_available_bytes:maia1{project_id!=""}
+
+    - record: netapp_volume_used_percentage:maia
+      expr: netapp_volume_used_percentage:maia2 or netapp_volume_used_percentage:maia1{project_id!=""}
+
+    - record: netapp_volume_snapshot_reserved_bytes:maia
+      expr: netapp_volume_snapshot_reserved_bytes:maia2 or netapp_volume_snapshot_reserved_bytes:maia1{project_id!=""}
+
+    - record: netapp_volume_snapshot_available_bytes:maia
+      expr: netapp_volume_snapshot_available_bytes:maia2 or netapp_volume_snapshot_available_bytes:maia1{project_id!=""}
+
+    - record: netapp_volume_snapshot_used_bytes:maia
+      expr: netapp_volume_snapshot_used_bytes:maia2 or netapp_volume_snapshot_used_bytes:maia1{project_id!=""}
+
+    - record: netapp_volume_inode_files_total:maia
+      expr: netapp_volume_inode_files_total:maia2 or netapp_volume_inode_files_total:maia1{project_id!=""}
+
+    - record: netapp_volume_inode_files_used:maia
+      expr: netapp_volume_inode_files_used:maia2 or netapp_volume_inode_files_used:maia1{project_id!=""}
+
+    - record: netapp_volume_inode_files_used_percentage:maia
+      expr: netapp_volume_inode_files_used_percentage:maia2 or netapp_volume_inode_files_used_percentage:maia1{project_id!=""}
+
+    - record: netapp_volume_total_ops:maia
+      expr: netapp_volume_total_ops:maia2 or netapp_volume_total_ops:maia1{project_id!=""}
+
```

Comment:
`netapp_volume_avg_latency_miliseconds:maia` is missing

Reply:
I wanted to replace it with 'netapp_volume_read_latency_milliseconds:maia` and `netapp_volume_write_latency_milliseconds:maia`. Avg latency alone is probably not very interesting.

Reply:
Makes sense. That means we need a documentation update